gnucash/report/html-utilities.scm

```diff
@@ -225,9 +226,7 @@
        comm-list)
       (gnc:html-table-set-col-headers!
        table (list (gnc:make-html-table-header-cell/size
-                    1 2 (if (null? (cdr comm-list))
-                            (G_ "Exchange rate")
-                            (G_ "Exchange rates"))))))
+                    1 2 (NG_ "Exchange rate" "Exchange rates" entries)))))
```

Comment:
In a normal context we would have `(NG_ "~a exchange rate" "~a exchange rates" entries)`, but this is the table header, right?

Anyway, by running `ninja pot` in gnucash.pot
```
#: gnucash/report/html-utilities.scm:229 gnucash/report/html-utilities.scm:247
msgid "Exchange rate"
msgstr ""

#: gnucash/report/html-utilities.scm:230 gnucash/report/html-utilities.scm:248
msgid "Exchange rates"
msgstr ""
```
gets replaced by
```
#: gnucash/report/html-utilities.scm:229 gnucash/report/html-utilities.scm:246
msgid "Exchange rate"
msgid_plural "Exchange rates"
msgstr[0] ""
msgstr[1] ""

```
To test it further, you will have to [`msgmerge`](https://wiki.gnucash.org/wiki/Translation#Update_an_existing_.po_file) the pot in your desired languages and translate the strings, update the .mo files…

Reply:
@fellen I've confirmed there's no guile/C error. The issue seems to be ngettext itself. The following logs the ngettext call:

```patch
modified   gnucash/report/html-utilities.scm
@@ -241,6 +241,7 @@
     (match comm-list
       (()
        (unless (zero? entries)
+         (pk 'entries entries (NG_ "Exchange rate" "plural" entries))
          (gnc:html-table-set-col-headers!
           table (list (gnc:make-html-table-header-cell/size
                        1 2 (NG_ "Exchange rate" "Exchange rates" entries)))))
```

And the logged output shows ngettext isn't pluralising at all.
`;;; (entries 2 "Exchange rate")`

Reply:
Addendum:

`LANG=C` gives `"Exchange rates"`
`LANG=en_GB.utf8`or `en_AU` or `de_DE` gives `Exchange rate`